includes/class-content-toc.php

```diff
@@ -195,11 +257,16 @@ public function generate_content_toc( $headers ) {
 
 				// Add anchor just before the matched header element
 				// Add filter to allow for custom anchor markup
+				// TODO: Do I still need a filter here?
 				$content = preg_replace(
 					'/' . preg_quote( $match_set[0], '/' ) . '/',
 					apply_filters(
```

Comment:
I wouldn't bother myself, especially given that this part is crucial functionality.